src/app/pages/datasets/components/dataset-details-panel/dataset-details-panel.component.ts

```diff
@@ -39,7 +39,8 @@ export class DatasetDetailsPanelComponent implements OnInit {
     || this.datasetHasChildrenWithShares
     || !!this.dataset.smb_shares?.length
     || !!this.dataset.nfs_shares?.length
-    || !!this.dataset.iscsi_shares?.length;
+    || !!this.dataset.iscsi_shares?.length
+    || this.dataset.name.endsWith('ix-applications');
```

Comment:
It's time to extract `ix-applications` here, below and in roles cell to a `dataset.utils.ts`.